cli/src/utils/renderVersionWarning.ts

```diff
@@ -55,12 +55,14 @@ function checkForLatestVersion(): Promise<string> {
               resolve((JSON.parse(body) as DistTagsBody).latest);
             });
           } else {
+            // eslint-disable-next-line @typescript-eslint/prefer-promise-reject-errors
             reject();
           }
         }
       )
       .on("error", () => {
         // logger.error("Unable to check for latest version.");
+        // eslint-disable-next-line @typescript-eslint/prefer-promise-reject-errors
         reject();
```

Comment:
This is the first time I can remember seeing `reject()` called with no argument. Interesting!

[`@typescript-eslint/prefer-promise-reject-errors`](https://typescript-eslint.io/rules/prefer-promise-reject-errors) was promoted to the [`strict-type-checked` config](https://v8--typescript-eslint.netlify.app/users/configs/#strict-type-checked) in v8.

Reply:
probably should reject with a descriptive error xd